### DISCUSSION TITLE
Gravitational redshift/time dilation question

### DISCUSSION THREAD

**Discussion by ExpEarth** [a-cosmology-group/acg#199] [Q&A] [answered]
I started a thread about this topic (#197) and I wanted to put the question more simply.

Suppose we have a light source positioned a distance _x_ from a mass.  Due to the presence of the mass, the light from the source is redshifted by an amount _z_ , as seen by a distant observer.  Now suppose a second mass is added at the same distance _x_ from the light source, but on the opposite side of it, such that the source is equidistant from the two masses.  Would the gravitational redshift and time dilation disappear, since the gravitational forces would now cancel out, or would they double to 2 _z_ ?

**Comment by Pierre314159** [comment, marked as answer]
The redshift would double.
The gravitational redshift depends on the difference in gravitational potential - not force - between observer’s and source’s locations. Potential is a scalar, not a vector, hence, there is no way one would cancel the other.

**Reply by ExpEarth** [reply]
Two opposite answers!

@sahil5d 

I have seen quite a few comments that agree with your answer.  Your suggestion reminded me of the question of what the redshift/time dilation would be at the centre of the Earth.  I just checked on this and found the question was asked in StackExchange: [https://physics.stackexchange.com/questions/10089/gravitational-time-dilation-at-the-earths-center](url)
The first answer given there would support Pierre's answer that it would double.

@Pierre314159 

Assuming you are correct, could you possibly describe how the redshift/time dilation would change in my discussion 197, either here or there?  For an observer at our position, which in my model is near the centre of a spherical shell of plasma, how would the redshift/time dilation in a light source change as you move inwardly from the shell towards the centre?  At first I thought it would be greatest near the shell, but now am not sure.  Also, could you comment on what the metric of the interior space would be?  With Sahil's interpretation I think the interior metric could be flat and Minkowski.  What would it be with your interpretation?

**Reply by RedshiftDrift** [reply]
Many inconsistent answers...

@ExpEarth the answer [on stackexchange](https://physics.stackexchange.com/q/65270) that says "Time would not be slower at center of earth" is 100% wrong.  Time at the centre of Earth is slower than time away from Earth.

@sahil5d is 80% wrong (because that was the certainty assigned to his answer).

As @Pierre314159 wrote (100% right), the difference in gravitational potential $U(\vec r)$ determines time-dilation/redshift, not the force $\vec F = -\vec\nabla U(\vec r)$.

To answer your question in [discussion 197](https://github.com/orgs/a-cosmology-group/discussions/197):
*"... the gravitational forces inside the shell would cancel out. Turning it back towards general relativity, however, would this mean that gravitational time dilation would not be seen in sources nearer the shell?"*  Redshift/time-dilation is not seen between an emitter and an observer located anywhere inside a spherical shell because the gravitational potential is constant, so there is no difference between different locations inside the shell.

*would "an observer at our position ... see gravitational time dilation in events nearer to the shell?"*  No, again: the potential is constant inside the spherical shell, so there is no observable time dilation for an observer inside the shell.

*"... due to the deeper gravitational potential well nearer the shell."* This is ***incorrect***.  Again, the gravitational potential near the shell (on the inside) is the same as the gravitational potential at the centre!

*"At first I thought it would be greatest near the shell, but now am not sure."*  Your first thought was incorrect.  The potential is the same everywhere inside the shell, there is no possible time dilation of light emitted anywhere from inside the shell that can be seen by an observer at the centre, according to relativity.

The metric inside the spherical shell is simply ***flat Euclidean/Minkowski***.

![potential of a shell](https://i.stack.imgur.com/kxw8g.png)

**Reply by ExpEarth** [reply]
@RedshiftDrift 

I'm satisfied that you and Pierre have the right answer.  However, I think you may be in error when you apply it to gravitational potentials within the spherical shell.  

> Redshift/time-dilation is not seen between an emitter and an observer located anywhere inside a spherical shell because the gravitational potential is constant, so there is no difference between different locations inside the shell.

You're saying the potential is constant everywhere, but is it not actually the gravitational forces that are constant everywhere, actually zero?  The forces inside are vectorial and all cancel each other.  Like many you seem to imply that the null gravitational force inside the sphere amounts to a constant potential everywhere inside and that the space inside is therefore Euclidian Minkowski.  The potentials are just scalars, however, and don't cancel in the same way.  Like you just said, by marking Pierre's answer as correct, is that they are additive.  So there must be a mathematical relationship for the potential at each point inside the sphere.  That's what I'm trying to find out, because it could be connected to the supernova time dilation in my model.

At the same time I would be happy if the metric inside is flat Minkowski.  That would be very static model.

**Reply by sahil5d** [reply]
Sorry about my previous post. The gravity index (I hope the idea of gravity as a refractive index gets developed) has more to do with the gravitational potential, not the net acceleration vector. Gravitational potential is scalar additive, and the distant observer will see a photon with twice the redshift.

@ExpEarth I wouldn't use "2z", because freq_obs / freq_emit = 1/(1+z), and I'd interpret doubling the redshift as
2 * (1 - freq_obs/freq_emit)
= 2 * (1 - 1/(1+z))
= 2 * z/(1+z)

Re the gravitational potential inside a shell, here's a toy sim with the scalar sum of the potential of 10,000 points, for a few test points in the interior, if you'd like to play with it.
https://codesandbox.io/p/sandbox/grav-potential-in-shell-fsyjcl

**Reply by ExpEarth** [reply]
@sahil5d 

Thanks for the correction on what a doubling of redshift means and for the toy simulator.  I think I'm better off not playing with it - I could hurt myself!

Here is a simple way of viewing the situation.  In my model, most of the universe's mass is in a spherical plasma shell situated near the Hubble radius.  For simplicity, we can consider the gravitational redshift that arises within the sphere at various points due only to two particles on opposite sides of the sphere.  Suppose the particles have total separation of 2 units, with the centre being at 1 unit from each.  At the centre the redshift due to one particle is _Gm/rc^2 = _Gm/c^2__  and for the combined redshift of both particles it would be twice this.  Now when the source is moved to a distance .1 unit from one of the particles along this axis, the gravitational redshift due to the nearby particle is magnified by 10 times for the observer at the origin compared to the previous situation.  The redshift due to the other particle at distance 1.9 units from the source is reduced to about half.   So the summed contributions to the redshift for a source at that position is about 10 times greater for the observer at the centre than when the source was at the centre also.  The gravitational redshift and time dilation would thus be greater for sources near the shell and fall off to a minimal value at the centre.

But how to do this in proper math for a 3D spherical shell?

**Reply by RedshiftDrift** [reply]
@ExpEarth 
> You're saying the potential is constant everywhere,

Correct
> but is it not actually the gravitational forces that are constant everywhere, actually zero?

The forces produced by each individual mass elements on the sphere are vectorial and **vary** as a function of position $\vec x$.  However the **total** gravitational force is exactly zero everywhere inside the sphere.
> The forces inside are vectorial and all cancel each other.

Correct
> Like many you seem to imply that the null gravitational force inside the sphere amounts to a constant potential everywhere inside and that the space inside is therefore Euclidian Minkowski.

Euclidean, yes
> The potentials are just scalars, however, and don't cancel in the same way.  Like you just said, by marking Pierre's answer as correct, is that they are additive.

Gravitational potentials never cancel.  They are always negative (and tend toward zero at large distances), and since they are additive, their sum cannot be zero.
> So there must be a mathematical relationship for the potential at each point inside the sphere.

There is.  The potential at a position $\vec x$ produced by a [mass distribution is](https://en.wikipedia.org/wiki/Gravitational_potential#Mathematical_form):

$$U(\vec x) = -G \int \frac{M(\vec r)}{|\vec r - \vec x|} dM(\vec r) \quad \mbox{ Eq. (1)}$$

For a total mass $M$ uniformly distributed on a sphere of radius $R$, Eq. (1) gives the mathematical relationship you are looking for:
$$U(\vec x) = -G \frac{M}{R}  \quad \mbox{ for any } \ \vec x \ \mbox{ inside the sphere,}$$

which is a constant everywhere inside the sphere, as claimed above.  (Note added later: this also agrees with @sahil5d's toy simuator - 10000 masses divided by R = 10 gives U = -1000.)  The gravitational-redshift/time-dilation depends on the difference between gravitational potentials $U(\vec x_1) - U(\vec x_2) = 0$.

Since $\vec F = -\vec\nabla U(\vec x)$ and since the derivative of a constant is zero, the force is zero for any $\vec x$ inside the sphere.

> That's what I'm trying to find out...

Newton derived the equation in 1687 and this is explained [everywhere on the web](https://en.wikipedia.org/wiki/Shell_theorem#), it shouldn't be difficult to find this equation.

From the last comment above
> At the centre the redshift due to one particle is Gm/rc^2 = Gm/c^2

This is incorrect.  The redshift is not an absolute quantity proportional to the potential but a difference in potential at two different location. Gm/c^2 is the redshift seen by an observer outside and far away from the spherical shell.

>  how to do this in proper math for a 3D spherical shell?
Use the solution of Eq. (1) given above.

**Reply by ExpEarth** [reply]
@RedshiftDrift 

Thanks, Louis.  This is a great help indeed!  I had seen lots of sites indicating that the forces cancel out inside the shell, including that Wiki page.  For the potentials inside the sphere I do not doubt your equation (1) is correct.  Unlike with the forces, though, I can't seem to get an intuitive feel for why that should be so.  The equation following (1) for the potential at any point inside the sphere - indicating a constant value - is quite remarkable.  This would truly give a flat Euclidean space inside the sphere, which is the observable universe in my model.

On this basis I would conclude that the time dilation in supernovae and quasars is NOT due to a gravitational redshift induced by shell baryons.  That leaves me still looking for the cause of the time dilation, but at least the metric is flat inside!

Concerning the last point, on the basis of everything that has come out recently from the JWST and elsewhere, would you say that a flat Euclidean or Minkowski space most accurately reflects what we are seeing at the remote distances?

**Reply by RedshiftDrift** [reply]
@ExpEarth 
For your "intuitive feel" to become useful, you first need to get rid of the **incorrect** impression that ~the force near a flat surface increases as you get closer to the surface~.

The gravitational force from a flat sheet [is independent of the distance](https://askfilo.com/physics-question-answers/a-there-is-an-infinite-thin-flat-sheet-with-mass-density-sigma-per-unit-area) above the sheet.

Intuitively then, the force toward the spherical cap (left on the image) does not increase as you get closer to the shell because it is approximately like a flat surface.

That force is balanced by the force from the rest of the sphere because that larger mass is distributed farther away.

---

As discussed in many posts on this forum, JWST data agrees with flat Minkowski space in which the redshift is produced by a "tired-light"-type interaction, e.g. [Fig. 5 in this article](https://www.mdpi.com/2075-4434/10/6/108).  A quantum derivation of the tired-light process shows that there is a delay between the interactions of different wavelengths with the electrons of the IGM.  This wavelength-dependent delay is responsible for the stretch of SNe light-curves, not time dilation.

![force-inside-sphere](https://github.com/a-cosmology-group/acg/assets/16748402/a6e4dad4-2569-4234-aa0c-a2f315f888da)

**Comment by ExpEarth** [comment]
Thanks, Louis, I've got it now!  I was thinking about the potentials incorrectly.  Thanks too for the reminder on the useful article by Lovyagin et al.
